src/components/carousel/DotsPagination.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC, useCallback } from 'react';
+import { EmblaCarouselType } from 'embla-carousel-react';
+
+import { useMultiStyleConfig } from '@chakra-ui/system';
+
+import Flex from '../flex';
+import Box from '../box';
+import DotsPaginationButton from './DotsPaginationButton';
+
+interface DotsPaginationProps {
+  currentSlideIndex: number;
+  numberOfSlides: number;
+  mainCarousel?: EmblaCarouselType;
+}
+
+const DotsPagination: FC<DotsPaginationProps> = ({
```

Comment:
I think you could refactor `NumbersPagination` to support both, wdyt? Something like
```typescript
   variant: "numbers" | "dots"
```

Reply:
Yea, I like that.